model/navigation/LtiNavigationService.php

```diff
@@ -146,6 +124,52 @@ protected function showThankyou(LtiLaunchData $launchData)
      */
     protected function getThankYouUrl()
     {
-        return _url('thankYou', 'DeliveryRunner', 'ltiDeliveryProvider');
+        return $this->getServiceLocator()->get(UrlHelper::class)->buildUrl('thankYou', 'DeliveryRunner', 'ltiDeliveryProvider');
```

Comment:
why thank you url is not configurable? and when you changed getUrl to buildUrl, don't we need to change getThankYouUrl to buildThankYouUrl

Reply:
you need to rename it, because in other places you've renamed same functions from get to build

Reply:
Done